insights/parsers/sysconfig_network.py

```diff
@@ -0,0 +1,29 @@
+"""
+NetSysconfig -file ``/etc/sysconfig/network``
```

Comment:
Is the name `NetworkSysconfig` a bit better?

Reply:
Done